yaksh/urls.py

```diff
@@ -100,7 +100,7 @@
             views.regrade, name='regrade'),
     url(r'^manage/regrade/paper/(?P<course_id>\d+)/(?P<answerpaper_id>\d+)/$',
             views.regrade, name='regrade'),
-    url(r'^manage/(?P<mode>[\w\-]+)/(?P<quiz_id>\d+)/$', views.test_quiz),
+    url(r'^manage/(?:godmode|usermode)/(?P<quiz_id>\d+)/$', views.test_quiz),
```

Comment:
This will not work!
Because "mode" argument is required by the views.
Kindly fix this and test once.